docs/source/quickstart/cli_yaml.rst

```diff
@@ -0,0 +1,178 @@
+    and optimize the hyperparameters using nested cross-validation. We can set the parameters for the outer 'assessment' and
+    inner 'selection' cross-validation loops.
+
+The complete YAML specification for this analysis is shown below and can be downloaded here: :download:`quickstart.yaml <../_static/files/quickstart/cli/quickstart.yaml>`.
+Make sure to change :code:`path/to/repertoires/` and :code:`path/to/metadata.csv` to the actual paths to the data on your local machine.
+
+.. highlight:: yaml
+.. code-block:: yaml
+
+    definitions:
+      datasets:
+        my_dataset: # user-defined dataset name
+          format: AIRR
+          params:
+            is_repertoire: true # we are importing a repertoire dataset
+            path: path/to/repertoires/          # path to the folder containing the repertoire .tsv files
+            metadata_file: path/to/metadata.csv
+
+      encodings:
+        my_kmer_frequency: # user-defined encoding name
+          KmerFrequency:   # encoding type
+            k: 3           # encoding parameters
+
+      ml_methods:
+        my_logistic_regression: LogisticRegression # user-defined ML model name: ML model type (no user-specified parameters)
+
+      reports:
+        my_coefficients: Coefficients # user-defined report name: report type (no user-specified parameters)
+
+    instructions:
+      my_training_instruction: # user-defined instruction name
+        type: TrainMLModel
+
+        dataset: my_dataset # use the same dataset name as in definitions
+        labels:
+        - signal_disease    # use a label available in the metadata.csv file
+
+        settings: # which combinations of ML settings to run
+        - encoding: my_kmer_frequency
+          ml_method: my_logistic_regression
+
+        assessment: # parameters in the assessment (outer) cross-validation loop
+          reports:  # plot the coefficients for the trained model
+            models:
+            - my_coefficients
+          split_strategy: random   # how to split the data - here: split randomly
+          split_count: 1           # how many times (here once - just to train and test)
+          training_percentage: 0.7 # use 70% of the data for training
+
+        selection: # parameters in the selection (inner) cross-validation loop
+          split_strategy: random
+          split_count: 1
+          training_percentage: 1 # use all data for training
+
+        optimization_metric: balanced_accuracy # the metric to optimize during nested cross-validation when comparing multiple models
+        metrics: # other metrics to compute for reference
+        - auc
+        - precision
+        - recall
+
+        strategy: GridSearch # strategy for hyperparameter optimization, GridSearch is currently the only available option
+
+        reports: []                # optional train ML model reports to run
```

Comment:
if it would simplify the quickstart, this now has a default value so it can be omitted, same as the three options below and strategy param :) if you think it should be there, just ignore this :)